.github/workflows/rust.yml

```
@@ -108,6 +108,49 @@ jobs:
+        components: clippy
+    - uses: Swatinem/rust-cache@v2
+      with:
+        key: hardware-${{ matrix.adapter }}
+    - name: Check
+      run: |
+        FEATURES="${{ matrix.features }}"
+        ARGS=""
+        [ -n "$FEATURES" ] && ARGS="--features $FEATURES"
+        cargo check --manifest-path="${{ matrix.manifest }}" --target thumbv7em-none-eabihf $ARGS
+    - name: Clippy
+      run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:4:86: Double quote to prevent globbing and word splitting

(shellcheck)